src/lib/temple/beacon.test.ts

```diff
@@ -0,0 +1,80 @@
+// import { crypto_sign_ed25519_pk_to_curve25519, crypto_sign_ed25519_sk_to_curve25519 } from 'libsodium-wrappers';
+import { browser } from 'webextension-polyfill-ts';
+
+import {
+  // createCryptoBox,
+  fromHex,
+  getDAppPublicKey,
+  getOrCreateKeyPair,
+  removeDAppPublicKey,
+  saveDAppPublicKey,
+  toHex,
+  toPubKeyStorageKey
+} from './beacon';
+
+const SAMPLE_PAYLOAD = 'hello, world!';
+
+describe('Beacon tests', () => {
+  describe('Storage', () => {
+    it('Format public key to storage key', () => {
+      const storageKey = toPubKeyStorageKey('myAwesomeDappKey');
+      expect(storageKey).toBe('beacon_myAwesomeDappKey_pubkey');
+    });
+    it('Stored key to be not empty', async () => {
+      await browser.storage.local.set({ beacon_something_pubkey: 'something' });
+      const dappKey = await getDAppPublicKey('something');
+      expect(dappKey).toBe('something');
+    });
+    it('Not stored key to be undefined', async () => {
+      const dappKey = await getDAppPublicKey('anotherKey');
+      expect(dappKey).toBe(undefined);
+    });
+    it('Stored key can be removed', async () => {
+      await browser.storage.local.set({ beacon_something_pubkey: 'something' });
+      await removeDAppPublicKey('something');
+      const dappKey = await getDAppPublicKey('something');
+      expect(dappKey).toBe(undefined);
```

Comment:
the same, you should test that `browser.storage.local.remove` mock was called with correct keys